Cargo.toml

```diff
@@ -23,7 +23,7 @@ split-debuginfo = "unpacked" # Faster debug builds on macOS
 # This results in roughly a 5% reduction in performance when compiling locally vs when
 # compiled via the CI pipeline.
 [profile.release]
-debug = false # Do not include debug symbols in the executable.
+debug = true # Do not include debug symbols in the executable.
```

Comment:
I sometimes think that we should include debug info by default and strip in the release step. Might be surprising to users that compile their own vector.

Reply:
Yeah, I kind of agree.

Maybe we should bite the bullet and use custom Cargo profiles to have a more debug-friendly release profile and then our "versioned release" optimized profile.

Reply:
Oh right! Totally forgot that there's custom profiles now. 

Yes, I'd be fully in support of doing that.